components/TripForm/index.js

```diff
@@ -114,7 +120,7 @@ export default function TripForm() {
 
         <StyledDIVAddTrip>
           <StyledlabelAddTrip htmlFor="description">
-            Description (max. 150 characters)*
+            Description 150/<span>{Letters}</span> characters left*
```

Comment:
I don't know if it's different in English, but I would expect the numbers to be the other way round. 🤔

 ```suggestion
            Description <span>{Letters}</span>/150 characters left*
```

Reply:
Thanks for the advice we changed the display.